datafusion/functions-array/src/udf.rs

```diff
@@ -392,3 +397,259 @@ impl ScalarUDFImpl for ArrayNdims {
         &self.aliases
     }
 }
+
+make_udf_function!(
+    ArrayAppend,
+    array_append,
+    array element,                                         // arg name
+    "appends an element to the end of an array.", // doc
+    array_append_udf                              // internal function name
```

Comment:
```suggestion
    array element,
    "appends an element to the end of an array.",
    array_append_udf
```

Reply:
I believe that we can get rid of these comments since the structure is apparent.